app/scripts/services/authorization.js

```diff
@@ -96,16 +96,22 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
+    var checkRulesForResourceVerb = function(rules, resource, verb) {
+      return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+    };
+
     var canI = function(verb, resource, projectName) {
```

Comment:
apiGroup is already part of the rules, to be more specific each key of the `rules` is constructed by `<apiGroup>/<resource>` and its value is an array of verbs